less/navigation.less

```diff
@@ -1 +1,70 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.main-navigation {
+    width: 100%;
+    height: 100px;
+    background-color: @nav-bg;
+    position: fixed;
+    opacity: 0.95;
+    
+
+    .nav-container {
+        height: 100%;
```

Comment:
try not to use percentages with height - it works fine here but a good rule of thumb is to stick with px or rem